src/styles/button.scss

```diff
@@ -0,0 +1,25 @@
+.buy {
+  justify-content: center;
+  box-sizing: border-box;
+  align-items: center;
+  text-transform: uppercase;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
Not fixed

Reply:
Still not fixed